doc/plugins.rst

```diff
@@ -5,73 +5,89 @@ tox plugins
 
 .. versionadded:: 2.0
 
-With tox-2.0 a few aspects of tox running can be experimentally modified
-by writing hook functions.  The list of of available hook function is
-to grow over time on a per-need basis.
+A growing number of `pluggy`_ hooks make tox extendable by writing plugins.
 
 
-writing a setuptools entrypoints plugin
+Writing a setuptools entrypoints plugin
 ---------------------------------------
 
-If you have a ``tox_MYPLUGIN.py`` module you could use the following
-rough ``setup.py`` to make it into a package which you can upload to the
-Python packaging index:
+You can create a new tox plugin with all bells and whistles via `Cookiecutter`_ (see `cookiecutter-tox-plugin <https://github.com/tox-dev/cookiecutter-tox-plugin>`_):
```

Comment:
`with all bells` should be `with all the bells`

Reply:
fixed.